src/lib.rs

```diff
@@ -203,6 +200,135 @@ mod zerocopy {
     pub(crate) use crate::*;
 }
 
+/// When performing a byte-slice-to-type cast, is the type taken from the prefix
+/// of the byte slice or from the suffix of the byte slice?
+#[doc(hidden)]
+#[allow(missing_debug_implementations, missing_copy_implementations)]
+pub enum CastType {
+    Prefix,
+    Suffix,
+}
```

Comment:
This should be removed; it was added for a feature that's since been removed from this PR.